modules/express/src/lightning/lightningWalletRoutes.ts

```diff
@@ -0,0 +1,41 @@
+import * as express from 'express';
+import { ApiResponseError } from '../errors';
+import { getLightningWallet, InvoiceQuery, UpdateLightningWalletSignedRequest } from '@bitgo/abstract-lightning';
+import { decodeOrElse } from '@bitgo/sdk-core';
+import * as t from 'io-ts';
+
+const UpdateWalletRequest = t.intersection([
+  UpdateLightningWalletSignedRequest,
```

Comment:
This should contain `signerTlsKey` instead of `encryptedSignerTlsKey`.  `handleUpdateLightningWalletCoinSpecific` 
 should encrypt `signerTlsKey` to `encryptedSignerTlsKey` using wallet passphrase (similar to `signerAdminMacaroon` -> `encryptedSignerAdminMacaroon`)